sympy/algebras/quaternion.py

```diff
@@ -757,3 +758,405 @@ def to_rotation_matrix(self, v=None):
 
             return Matrix([[m00, m01, m02, m03], [m10, m11, m12, m13],
                           [m20, m21, m22, m23], [m30, m31, m32, m33]])
+
+    def scalar_part(self):
+        r"""Returns scalar part($\mathbf{S}(q)$) of the quaternion q.
+
+        Explanation
+        ===========
+
+        Given a quaternion $q = a + bi + cj + dk$ and returns $\mathbf{S}(q) = a$
+
+        Examples
+        ========
+
+        >>> from sympy.algebras.quaternion import Quaternion
+        >>> q = Quaternion(4, 8, 13, 12)
+        >>> q.scalar_part()
+        4
+
+        """
+
+        return self.a
+
+    def vector_part(self):
+        r"""
+        Returns vector part($\mathbf{V}(q)$) of the quaternion q.
+
+        Explanation
+        ===========
+
+        Given a quaternion $q = a + bi + cj + dk$ and returns $\mathbf{V}(q) = bi + cj + dk$
+
+        Examples
+        ========
+
+        >>> from sympy.algebras.quaternion import Quaternion
+        >>> q = Quaternion(1, 1, 1, 1)
+        >>> q.vector_part()
+        0 + 1*i + 1*j + 1*k
+
+        >>> q = Quaternion(4, 8, 13, 12)
+        >>> q.vector_part()
+        0 + 8*i + 13*j + 12*k
+
+        """
+
+        return Quaternion(0, self.b, self.c, self.d)
+
+    def axis(self):
+        r"""
+        Returns the axis($\mathbf{Ax}(q)$) of the quaternion.
+
+        Explanation
+        ===========
+
+        Given a quaternion $q = a + bi + cj + dk$ and returns $\mathbf{Ax}(q)$ ie normalized versor (vector part) of that quaternion.
```

Comment:
The "ie normalized versor (vector part)" isn't correct. The versor and the vector part are distinct concepts, not an explanation of the other. A solution would be: *[...], i.e., the versor of the vector part of that quaternion*.

The equation is `$\mathbf{Ax}(q) = \mathbf{U}[\mathbf{V}(q)]$`.